src/extractors/custom/phpspot.org/index.js

```diff
@@ -0,0 +1,25 @@
+export const PhpspotOrgExtractor = {
+  domain: 'phpspot.org',
+
+  title: {
+    selectors: ['h3.hl'],
+  },
+
+  author: null,
+
+  date_published: null,
```

Comment:
Could you please test querying the date using something like the following code:
```javascript
  date_published: {
    selectors: ['h4.hl'],
    format: 'YYYY年MM月DD日',
  },
```

Reply:
Thank you! I fixed.